### PR TITLE
CLOUD-811 added paymentSessionId as optional query param to list intr…

### PR DESCRIPTION
**Pull request by lohithv**

### CODE REVIEW

**Review by mchopra-woolworths — APPROVED** [review]

**Review by mchopra-woolworths — CHANGES_REQUESTED** [review]
1.0.7 is already in master. New version has to be created with 1.0.8

File: src/wpay-1.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "info":{
     "title":"Wpay APIs",
     "description":"Wpay APIs for payments and gifting",
-    "version":"1.0.6"
+    "version":"1.0.7"
```
Comment (mchopra-woolworths) [inline src/wpay-1.json:6]: @lohithv - Hey Lohith, there is already a version of 1.0.7 in master. Please uplift this change to 1.0.8

Comment (lohithv) [reply src/wpay-1.json]: @mchopra-woolworths done

**Review by JamesMBligh — CHANGES_REQUESTED** [review]
v1.0.7 has been amended since this PR was raised to deal with some EDPay changes.  This won't affect wpay-1.json as it will be merged but it will affect wpay-1-0-8.json as the changes won't be merged.

Could you back merge from master then we can approve the PR.

**Comment by lohithv** [comment]
> v1.0.7 has been amended since this PR was raised to deal with some EDPay changes. This won't affect wpay-1.json as it will be merged but it will affect wpay-1-0-8.json as the changes won't be merged.
> 
> Could you back merge from master then we can approve the PR.

@JamesMBligh done

**Pull request merged**